src/views/v2/widgets/events-list.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Widget: Events List
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var array    $events             The array containing the events.
+ * @var string   $rest_url           The REST URL.
+ * @var string   $rest_nonce         The REST nonce.
+ * @var int      $should_manage_url  int containing if it should manage the URL.
+ * @var string[] $container_classes  Classes used for the container of the view.
+ * @var array    $container_data     An additional set of container `data` attributes.
+ * @var string   $breakpoint_pointer String we use as pointer to the current view we are setting up with breakpoints.
+ * @var array    $messages           An array of user-facing messages, managed by the View.
+ */
+
+$events = [
+
+];
+$rest_url = 'https://tri.be/';
+$rest_nonce = 'a3ghv98awe98';
+$should_manage_url = '0';
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 9 spaces but found 1 space

<sub>hash: 6e97355dfa563c8f3e39696eeba8b624</sub>